package.json

```diff
@@ -84,7 +84,6 @@
     "save-build-info": "echo \\{\\\"gitRevision\\\": \\\"$(git rev-parse HEAD)\\\", \\\"buildTimestamp\\\": $(date -u '+%s000')\\} > public/build-info.json"
   },
   "devDependencies": {
-    "setimmediate": "^1.0.5",
```

Comment:
Unrelated. It looks like #4155 manually edited this list. yarn sorts the dependencies automatically.